__tests__/simple-send-tx.test.js

```diff
@@ -137,17 +137,26 @@ describe('simple-send-tx api', () => {
   });
 
   it('should accept a custom token transaction (token as string)', async () => {
+    // Since the loaded wallet is the same in all tests
```

Comment:
Why was it working before?

Reply:
Because we used to mark the `selected_as_input` on the history object, which gets "reloaded" from the fixture without the `selected_as_input` or the "spent_by" from the previous test.
Since we removed the `selected_as_input` to a separate collection it does not get "auto cleaned".

I used the new wallet but i can change to just un`select_as_input` on the previous test, what do you think?